### PR TITLE
MOSIP-36413 Removed childauth filter

### PR DESCRIPTION
**Pull request by kameshsr**

### CODE REVIEW

File: authentication/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# This repo is moved to [Id-authentication](https://github.com/mosip/id-authentication/tree/develop) as part of https://mosip.atlassian.net/browse/MOSIP-36413
```
Comment (ashok-ksharma) [inline authentication/README.md:1]: 1. Let's not add the reference to develop branch. We can directly mention the repo and it's sub repo where it has been moved.
2. Let's remove the JIRA number as well.
3. Line has been commented. We should remove the comment.

File: authentication/authentication-childauthfilter-impl/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# This repo is moved to [Id-authentication](https://github.com/mosip/id-authentication/tree/develop) as part of https://mosip.atlassian.net/browse/MOSIP-36413
```
Comment (ashok-ksharma) [inline authentication/authentication-childauthfilter-impl/README.md:1]: 1. Let's not add the reference to develop branch. We can directly mention the repo and it's sub repo where it has been moved.
2. Let's remove the JIRA number as well.
3. Line has been commented. We should remove the comment.

**Review by ashok-ksharma — CHANGES_REQUESTED** [review]

File: authentication/README.md (hunk 1 above)
Comment (kameshsr) [reply authentication/README.md]: udated

File: authentication/authentication-childauthfilter-impl/README.md (hunk 2 above)
Comment (kameshsr) [reply authentication/authentication-childauthfilter-impl/README.md]: updated

**Review by ashok-ksharma — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**